AuthenticationAndAuthorization/BusinessLogic/Services/UserServices.cs

```diff
@@ -0,0 +1,46 @@
+﻿using BusinessLogic;
+using BusinessLogic.Model;
+using DataAccounts;
+using DataAccounts.Repositories;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+
+namespace BusinessLogic
+{
+    public class UserService: IUserService
+    {
+        private readonly IUserRepositories _userRepositories;
+        private readonly IJwtService _jwtService;
+        
+        public UserService(IUserRepositories userRepositories, IJwtService jwtService)
+        {
+            _userRepositories = userRepositories;
+            _jwtService = jwtService;
+        }
+
+        public async Task<string> LoginUserAsync(LoginUserRequest model)
+        {
+            var user = await _userRepositories.GetEmail(model.Email);
+            var result = await _userRepositories.IsCorrectPassword(model.Email,model.Password);
+            if (result == false)
+            {
+                throw new ArgumentException();
+            }
+            return _jwtService.GenerateJwt(user.Id,user.Username);
```

Comment:
Add spaces. We put space before return.
Also add some message to your exception. It's not clear what happened and how end user will understand what is wrong?
Also add global exception handler which will process exceptions happened and map them to bad request/500 error or something